### PR TITLE
update dpg and vibe

### PR DESCRIPTION
**Pull request by burner**

### CODE REVIEW

**Comment by burner** [comment]
could you turn this into a new release?

File: dub.sdl [hunk 1]
```diff
@@ -5,8 +5,8 @@ license "MIT"
 copyright "Copyright © 2016"
 targetType "sourceLibrary"
 
-dependency "dpq2" version="~>1.0.0-alpha.11"
-dependency "vibe-d:core" version="~>0.8.3"
+dependency "dpq2" version=">=1.0.5"
+dependency "vibe-d:core" version="~>0.8.4"
```
Comment (denizzzka) [inline dub.sdl]: Not agreed with this line: someone can use old vibe.d

File: dub.sdl [hunk 2]
```diff
@@ -22,7 +22,7 @@ configuration "integration_test_vibe_core" {
     targetType "executable"
     buildType "unittest"
     versions "IntegrationTest"
-    dependency "vibe-core" version="~>1.4.0"
+    dependency "vibe-core" version="~>1.4.5"
```
Comment (denizzzka) [inline dub.sdl]: Again, not agreed with this line: someone can use old vibe.d

File: dub.sdl (hunk 1 above)
Comment (burner) [reply dub.sdl]: so ">= 0.8.3" then?

**Comment by burner** [comment]
updated

File: dub.sdl (hunk 1 above)
Comment (denizzzka) [reply dub.sdl]: As I know, 0.8.4 also belongs inside of  ~>0.8.3, and 1.0.5 inside of ~>1.0.0-alpha.11

Comment (denizzzka) [reply dub.sdl]: My proposal is: change dpq2 to ~>1.0.0 and not touch other versions.

Comment (burner) [reply dub.sdl]: but my hope is to get at least 1.0.3 so I can use columnExists for Row structs

Comment (denizzzka) [reply dub.sdl]: vibe.d.db.postgresql package should get lates version of dpq2
also you can define any version of dpq2 in your dub config to override this version

**Comment by denizzzka** [comment]
> updated

does not look like what I said

**Comment by schveiguy** [comment]
@burner The `~>` designation is what you want. From code.dlang.org:

``Restrict to a certain minor version: "~>2.2.13", equivalent to ">=2.2.13 <2.3.0"``

And it should be 1.0.0 for dpq2, because that should cover 1.0.0 - 1.0.anything. It WILL get the latest minor version that's available. Just use `dub upgrade`.

`>=` is not correct, because then it would get 1.1.x also. You'd probably need a minor revision of this repository to accommodate any changes there.

**Comment by burner** [comment]
and another try

**Comment by burner** [comment]
thank you

**Pull request merged**